pkgs/by-name/qq/qqsp/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  adwaita-qt,
+  libadwaita,
+  adw-gtk3,
+  gst_all_1,
+  gnome-themes-extra,
+  librsvg,
+  qt5,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "qqsp";
+  version = "1.9";
+  src = fetchFromGitHub {
+    owner = "Sonnix1";
+    repo = "Qqsp";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-eDgoa+/dcJ8Ti+YLHgKUKus0+zRrFEuJ19wUpbFpcBU=";
+  };
+  buildInputs = [
+    adwaita-qt
+    libadwaita
+    adw-gtk3
+    gnome-themes-extra
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-ugly
+    qt5.qtmultimedia
+    qt5.qtwebengine
+    qt5.qtbase
+  ];
+  nativeBuildInputs = [
+    qt5.qmake
+    qt5.qtbase
+    qt5.qtmultimedia
+    qt5.qtwebengine
```

Comment:
No need to duplicate these
```suggestion
```

Reply:
If they are removed then the package fails to build when strictDeps is enabled